rustls/src/msgs/fragmenter.rs

```diff
@@ -8,22 +8,12 @@ pub const MAX_FRAGMENT_LEN: usize = 16384;
 pub const PACKET_OVERHEAD: usize = 1 + 2 + 2;
 pub const MAX_FRAGMENT_SIZE: usize = MAX_FRAGMENT_LEN + PACKET_OVERHEAD;
 
+#[derive(Default)]
```

Comment:
I think this needs to be a custom `Default` implementation that sets `max_frag` to `MAX_FRAGMENT_LEN` rather than `0`?